db/migrate/20240123014507_create_apps_evolution_apis.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAppsEvolutionApis < ActiveRecord::Migration[6.1]
+  def change
+    create_table :apps_evolution_apis do |t|
+      t.references :account, null: false, foreign_key: true
+      t.string :status
+      t.boolean :active
+      t.string :endpoint_url
+      t.string :token
+      t.string :phone
+      t.string :name
```

Comment:
O token habilitar o not null e deixar o valor padrão como string vazia e adicionar no model a validação para não poder ser vazia validate present